### PR TITLE
Initial code checking from crio

### PR DESCRIPTION
**Pull request by baude**
Did a little Makefile workery as well.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+include ../Makefile.inc
```
[the remark this answers is not in the document]
Comment (baude) [reply src/Makefile]: i was just trying to keep the structure the same.

[the remark this answers is not in the document]
Comment (baude) [reply src/Makefile]: fixed

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
Merging

**Pull request merged**